cmd/ndm_daemonset/probe/udevprobe.go

```diff
@@ -228,6 +229,15 @@ func (up *udevProbe) scan() error {
 					deviceDetails.DependentDevices = dependents
 					klog.Infof("Dependents of %s : %+v", deviceDetails.DevPath, dependents)
 				}
+				// the device type reported by udev will always be disk/partition. Using this info
```

Comment:
Device type can be loop, lvm, etc too, right? I think we should have a doc explaining all the device types and relationships What do you think?

Reply:
This is the device type that is reported by udev with the `DEVTYPE` variable. It will always be disk/partition.  
The types loop, lvm etc that we see is from sysfs which we use everywhere. The sysfs uses the udev reported type to perform its device type identification logic.